swagger.yaml

```diff
@@ -0,0 +1,198 @@
+swagger: "2.0"
+info:
+  description: "An API used to search information on published datasets by the ONS."
+  version: "1.0.0"
+  title: "Search API"
+  license:
+    name: "Open Government Licence v3.0"
+    url: "http://www.nationalarchives.gov.uk/doc/open-government-licence/version/3/"
+basePath: "/v1"
+tags:
+- name: "Public"
+- name: "Private user"
+schemes:
+- "http"
+parameters:
+  dimension_name:
+    name: name
+    description: "A dimension from a dataset."
+    in: path
+    required: true
+    type: string
+  edition:
+    name: edition
+    description: "An edition of a dataset"
+    in: path
+    required: true
+    type: string
+  id:
+    name: id
+    description: "Id that represents a dataset"
+    in: path
+    required: true
+    type: string
+  instance_id:
+    name: instance_id
+    description: "A unique id for an instance."
+    in: path
+    required: true
+    type: string
+  limit:
+    name: limit
+    description: "The number of resources to be returned."
+    in: query
+    type: integer
+  offset:
+    name: offset
+    description: "The first row of resources to retrieve, starting at 0. Use this parameter as a pagination mechanism along with the limit parameter."
+    in: query
+    type: integer
+  query:
+    name: q
+    description: "The query text to search datasets with."
+    in: query
+    type: string
+    required: true
+  version:
+    name: version
+    description: "A version of a dataset"
+    in: path
+    required: true
+    type: string
+securityDefinitions:
+  FlorenceAPIKey:
+    name: florence-token
+    description: "API key used to allow florence users to create and query the progress of importing a dataset."
```

Comment:
Do we actually need a Florence Key? If its just for the preview, then this might not be the right description

Reply:
It is just for preview